### PR TITLE
Improve DMA CHAIN_TO docs

### PR DESCRIPTION
**Pull request by 9names**
This is in response to issue #56.

Note that the documentation that this is replacing states precisely the behaviour that the PAC exhibits, so this isn't really "fixing" the documentation as much as trying to spell it out a bit more clearly.

https://docs.rs/rp2040-pac/latest/rp2040_pac/dma/ch/ch_al2_ctrl/struct.W.html#method.chain_to

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "rp2040-pac"
-version = "0.3.0"
+version = "0.3.1"
```
Comment (thejpster) [inline Cargo.toml:3]: Should be in a separate PR.

Comment (9names) [reply Cargo.toml]: I've amended the last commit to not bump the version.

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - None
 
+## [0.3.1] [Crates.io](https://crates.io/crates/rp2040-pac/0.3.1) [Github](https://github.com/rp-rs/rp2040-pac/releases/tag/v0.3.1)
```
[the remark this answers is not in the document]
Comment (9names) [reply CHANGELOG.md]: I've amended the last commit so this is now under # Unreleased instead of 0.3.1

File: svd/rp2040.yaml [hunk 3]
```diff
@@ -1,6 +1,26 @@
 _svd: "./rp2040.svd"
 
 DMA:
+  CH0_CTRL_TRIG:
+    _modify:
+      "CHAIN_TO":
+        description: "When this channel completes, it will trigger the channel indicated by CHAIN_TO. Disable by setting CHAIN_TO = _(this channel)_. \\n
```
[the remark this answers is not in the document]
Comment (thejpster) [reply svd/rp2040.yaml:7]: Any thoughts on this one?

[the remark this answers is not in the document]
Comment (9names) [reply svd/rp2040.yaml:7]: Each channel? No, not any way that I'm aware of - there is only one cluster left in the output SVD, so the description is shared with all of them.
Would be happy to be wrong about this.

**Review by jannic — APPROVED** [review]
